api service integration changes (#5958)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/openidconnect/add-tester.md

#### Add to Tester

> **Note:** The OIN Submission Tester only supports SSO integrations. The **Add to Tester** option isn't available for SCIM integrations.

* Click **Add to Tester** next to the instance in the **Application instances for testing** list. This includes it for testing with the OIN Submission Tester. The **Add to Tester** option only appears for SSO instances that are active and eligible for testing.

    The corresponding test cases are populated with the instance name and the **Run test** option is enabled in the OIN Submission Tester.

* Click **Remove from Tester** to disable the test cases that are associated with the SSO instance.

>     The instance name and test results are removed for the corresponding test cases in the OIN Submission Tester. The **Run test** option is also disabled.

Reviewer: ```suggestion
    The app name and test results are removed from the corresponding test cases in the OIN Submission Tester. The **Run test** option is also disabled.
```
